.github/ISSUE_TEMPLATE/general_issue.yml

```diff
@@ -0,0 +1,42 @@
+name: General Issue
+description: Report an issue unrelated to Saving
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please include as much information as possible.
+  - type: checkboxes
+    id: renderer
+    attributes:
+      label: Renderer
+      description: Which renderer does this issue occur on?  If you are unsure, you can check the renderer in the Metadata Editor.
```

Comment:
Can we add something like "(click the 🛈 button at the top of the text editor)"? Lots of users have no idea there is a metadata panel.

Also, the button popup calls it "Properties", so we should call it that here.

Reply:
done